src/Router.jsx

```diff
@@ -40,11 +52,15 @@ export const Router = () => {
 
                     <Route
                         exact
-                        path={`${routes.Profile}/:id`}
-                        element={<ProfilePage />}
+                        path={`${routes.Post}/:id`}
+                        element={<PostDetailsPage />}
                     />
 
-                    <Route exact path={routes.Chat} element={<ChatPage />} />
+                    <Route
+                        exact
+                        path={`${routes.Chat}/:id`}
```

Comment:
Chat route should take id?

Reply:
yup, we have multiple rooms implemented on firebase